dns.ks.in

```diff
@@ -15,8 +15,14 @@ if [ "@KSTEST_OS_NAME@" == "rhel" ]; then
     # rhbz#1989472
     check_resolv_conf_is_by_nm
 else
-    # rhbz#2018913
-    check_resolv_conf_is_by_resolved
+    # rhbz#2018913: creating of the symlink by systemd-resolved
+    # is broken, but it does not seem to be a big deal for installer.
+    # Non-existing /etc/resolv.conf is ok as well, the symlink will
+    # be created during boot.
+    #check_resolv_conf_is_by_resolved
```

Comment:
The commented-out function is no longer used anywhere...

Reply:
I would remove the function if I removed the call, but I just commented it out because we may well have to comment it in it later if/when the BZ is fixed :(